### ISSUE TITLE
The Incredible Adventures of Van Helsing II (272470)

### ISSUE THREAD

**Issue by stratus-ss** [ValveSoftware/Proton#564]
The game loads and you can start or resume previous games. However the UI elements are all white which means you cannot see, or change your options or view the map

Steam System Info:
https://gist.github.com/stratus-ss/73e90ae7dcc42caf324190ea36e979cc

Two very interesting notes are  that

1) If the steam controller is plugged in when the game is launched, the UI cannot be navigated and the game must be forcibly killed

2) The game runs fine with no graphical anomilies that I can see with this version of wine: wine-staging-pba-git 3.13.r14.g12d22ddf+wine.3.13.r0.g25cc380b8e+pba.3.13.r68.68129ed-1

Without building from source, how do I launch a game with proton such that I can generate useful information? strace provides some clues but there must be a better way to get more detailed info

**Comment by diraven** [comment]
The configuration file is called user_settings.py and it is located in the "Proton 3.7" directory. Rename user_settings.sample.py to user_settings.py, then try again. Look for the log files in your $HOME directory.

How to find out where Proton is installed: https://github.com/ValveSoftware/Proton/wiki/Proton-FAQ

**Comment by stratus-ss** [comment]
Log gist: https://gist.github.com/stratus-ss/6c4e6438cda5ca7e72c211667ccb4ed3

Arch Linux 64-bit (4.18.4-arch1-1-ARCH)
nvidia 396.54-1
i7-4790K/GTX 970

**Comment by diraven** [comment]
Can you execute `ulimit -Hn` and check what number does it return?

**Comment by stratus-ss** [comment]
4096

**Comment by diraven** [comment]
Ok, then either try to (temporarily) raise the limits: https://github.com/zfigura/wine/blob/esync/README.esync

Or disable esync alltogether for the game by setting `PROTON_NO_ESYNC=1 %command%` as launch options for the game.

**Comment by stratus-ss** [comment]
Thanks for the link. Funny thing is when i am troubleshooting something for work I look at the limits.conf, I've never had to tinker with it on my desktop

Setting the launch option resolves the last of the issues i have seen with Helsing II

I am going to keep this open until someone official closes it. I still think that the experience needs to be handled for the average player

**Comment by diraven** [comment]
So is everything works as intended with esync turned off?

**Comment by stratus-ss** [comment]
Yes, I ran a few minutes of single player without glitches and I also launched a multiplayer.

It seems to have resolved the issue with the steam controller as well. 

The game still crashes on EXIT but that is very minor as the user is quitting anyways

**Comment by nstgc** [comment]
Have any of you tried the first game? I haven't been able to get it to run even with noesync=1.

**Comment by stratus-ss** [comment]
The first game worked flawless for me (except for the same issue with the controller). If I removed the controller I had 0 issues with Helsing I

**Comment by nstgc** [comment]
Did you do anything special? All I've been getting are CTDs at launch. I've tried forcing DX11, forcing it off, forcine noesync. . .

https://gist.github.com/nstgc/02457d7920931a3f8d69cc7b9fa41a2c

**Comment by stratus-ss** [comment]
I'm using proton 3,7-4 beta if that helps

**Comment by nstgc** [comment]
It does. I haven't tried the beta yet with Helsing 1.

There are just SO many Linux games in my library now! Its hard to test them all :grinning:

**Comment by diraven** [comment]
If you have issues with the game - check your logs, or post them via gist just like it's done above. Should help a lot in finding out the issue.

**Comment by nstgc** [comment]
@diraven You mean like I've been doing? ie: https://github.com/ValveSoftware/Proton/issues/676

**Comment by diraven** [comment]
Yes. I did not see the log for the first game you mentioned. Might be a good idea to make new issue for it with all the necessary data.

**Comment by nstgc** [comment]
I wasn't ready to make an issue, but since people were discussion the first game, I thought I may as well chime in. I put more effort into opening a new issue than joining an existing one. Whether that is right or wrong, I don't know, but its the way I tend to do things.

**Comment by diraven** [comment]
This issue is about Helsing II. If you like to discuss Helsing I - it's better to do it in a separate issue.

**Comment by nstgc** [comment]
Which, again, is what I did: https://github.com/ValveSoftware/Proton/issues/599#issuecomment-416002693
